### PR TITLE
guidelines to contribute

### PR DESCRIPTION
**Pull request by gennaro-tedesco**
I have drafted a minimal `CONTRIBUTING.md` that briefly describes guidelines for independent contribution and provides boilerplate template, following [this example](https://github.com/lucax88x/nvim-awesome/pull/21/files).

I am not sure this is the format you intended; if not please point me to the right direction and I will amend accordingly. Where exactly are you storing the list of plugins to populate the interface with (because it is unclear whether for requests one wants to _merge_ PR - merge into where)?

This project is great and I hope it can grow consistently!

### CODE REVIEW

File: data/plugins/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## How to contribute
+
+If you wish to recommend a plugin or framework not included in the list yet:
+
+1. create a `.json` file in [data/plugins/myplugin.json](myplugin.json) including plugins credentials as
```
Comment (lucax88x) [inline data/plugins/CONTRIBUTING.md]: as a general rule I am using owner-repository.json as fileName, just to ensure in the future we don't have 2 plugins with same name

File: data/plugins/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "tags": [
+    "awesomeness",
+    "coolness",
+    "lua"
+  ],
+  "examples": []
+}
+```
+
+2. (optional) (make sure that the repository fulfils the below criteria)
```
Comment (lucax88x) [inline data/plugins/CONTRIBUTING.md]: Only criteria I would add is to just check existing tags

**Comment by lucax88x** [comment]
Thanks! Much appreciated :)

**Review by lucax88x — CHANGES_REQUESTED** [review]

File: data/plugins/CONTRIBUTING.md (hunk 1 above)
Comment (gennaro-tedesco) [reply data/plugins/CONTRIBUTING.md]: Of course! Fixed in af87fabbc3e7fef4f068b17fa5e00b879ceff55f

File: data/plugins/CONTRIBUTING.md (hunk 2 above)
Comment (gennaro-tedesco) [reply data/plugins/CONTRIBUTING.md]: Added this comment in af87fabbc3e7fef4f068b17fa5e00b879ceff55f

**Pull request merged**